tests/SprykerSdkTest/Integrator/IntegratorFacadeTest.php

```diff
@@ -154,6 +154,8 @@ public function testRunInstallationWireConsole(): void
     }
 
     /**
+     * @group test1
```

Comment:
I think we don't need it here. If we want to use the groups, let's give them more meaningful names